index.js

```diff
@@ -70,6 +100,47 @@ async function run() {
       })
     }
 
+    if (environmentSecrets) {
+
+      // If environment array is missing, create it
+      if (!Array.isArray(containerDef.secrets)) {
+        containerDef.secrets = [];
+      }
+
+      // Get pairs by splitting on newlines
+      environmentSecrets.split('\n').forEach(function (line) {
+        // Trim whitespace
+        const trimmedLine = line.trim();
+        // Skip if empty
+        if (trimmedLine.length === 0) { return; }
+        // Split on =
+        const separatorIdx = trimmedLine.indexOf("=");
+        // If there's nowhere to split
+        if (separatorIdx === -1) {
+            throw new Error(`Cannot parse the environment secret '${trimmedLine}'. Environment secret pairs must be of the form KEY=value.`);
+        }
+        // Build object
+        const secret = {
+          name: trimmedLine.substring(0, separatorIdx),
+          valueFrom: trimmedLine.substring(separatorIdx + 1),
+        };
+
+        // Search container definition environment for one matching name
+        const variableDef = containerDef.secrets.find((e) => e.name == secret.name);
+        if (variableDef) {
+          // If found, update
+          variableDef.valueFrom = secret.valueFrom;
+        } else {
+          // Else, create
+          containerDef.secrets.push(secret);
+        }
+      })
+    }
+
+    if (awslogsGroup && awslogsRegion && containerDef.logConfiguration && containerDef.logConfiguration.options) {
+      containerDef.logConfiguration.options["awslogs-group"] = awslogsGroup;
+      containerDef.logConfiguration.options["awslogs-region"] = awslogsRegion;
```

Comment:
Nit; It might also be worthwhile to also include the option to set `awslogs-stream-prefix` and `awslogs-create-group` as well because that'll ship the complete set of AWS logs options for customers.